nixos/modules/services/networking/firewall.nix

```
@@ -546,7 +546,7 @@ in
       options nf_conntrack nf_conntrack_helper=1
     '';
 
-    assertions = [ { assertion = cfg.checkReversePath -> kernelHasRPFilter;
+    assertions = [ { assertion = (cfg.checkReversePath != false) -> kernelHasRPFilter;
```

Do you think it would be more readable to say?

```suggestion
    assertions = [ { assertion = cfg.checkReversePath == false || kernelHasRPFilter;
```

---

Yeah, I think that improves things. Having to read `!== false` not pleasant.